### PR TITLE
Removes the vanguard role

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Removes the vanguard role entirely and adds a surplus vanguard crate to cargo for 3800 credits with (nerfed) vanguard gear.

## How This Contributes To The Skyrat Roleplay Experience

- Full stop, effectively locking gateways behind 300 hours of gameplay (veteran crew) and 40 hours of sci (timelock) absolutely sucks. It locks a very large part of the game from newer players or those not able to make veteran crew, compared to the two other fairly-gimmick jobs that veteran crew gets (Blueshield, NT rep)
- It powercreeps our gateways by giving them a very strong starting point to have to start from, leading to it having to increase even further to ridiculous levels (see: late-game Mesa and Astrum) where nothing can kill the Vanguard that already start very strong.
- It prevents us from playing TG gateways in the intended balance, since the maps weren't designed for the people who enter with a large amount of armor, guns, and medicine right off the bat without setup time.
- (Personal opinion inbound) I prefer the aesthetic of five ragtag crew members going in with the scraps they made over a trained organization being deployed to the station to handle their problems for them.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

image

</details>

## Changelog

:cl:
del: The Vanguard job has been removed
add: Surplus (nerfed) Vanguard gear can be bought from a misc. crate in cargo for 3600 credits.
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Based af.

**Comment by MixMashy** [comment]
I disagree quite hard with this as mainly a lot of people get veterans for the roles and benefits. Becoming a bigger part of the community and being proven to be respectable and heavily RP-orientated. As for how TG intended their gates.. id rather awaits the rework and or change to gateways and how they perform as they are currently not in use properly, As for power creep. I don't see much wrong with the current kit if you're just gonna nerf it. Just nerf it but I don't see an issue with the current kit.. it really isn't strong at all. By the feel of it. Did a little bit of testing myself. I don't see a mega-corporation sending a ragtag group of suicidal civilians through a strange gateway full of unknowns. They'd have the professionalism and resources to send highly trained professionals. Makes NT look like a fool for not being able to train highly paid professionals as they do with Blueshield or NT Rep as you have said.

**Comment by Iamgoofball** [comment]
:+1:

AWAY TEAMS FOR LIFE

**Comment by Iamgoofball** [comment]
think about it:

star trek didn't keep Guy Who Goes Down To Planets around on call 24/7 to go down to the planet

they sent the captain, some redshirts, the CMO and doctors, and sometimes the CE and a team of engineers

as a TEAM

an AWAY TEAM

**Comment by Iamgoofball** [comment]
if it's good enough for motherfuckin STARFLEET its good enough for our boys in Nanotrasen

**Comment by Iamgoofball** [comment]
also as a /tg/ dev we absolutely design our gateways around multiple people interacting and exploring, it's very boring if people go in solo and murder everything with OP guns and leave

**Comment by Zonespace27** [comment]
> I disagree quite hard with this as mainly a lot of people get veterans for the roles and benefits.

I have stated before and I will state again: I do not care if people got vet crew for a very-very-rarely enabled role, and the existence of veteran crew _should not_ impede game design

> As for how TG intended their gates.. id rather awaits the rework and or change to gateways and how they perform as they are currently not in use properly,

They quite literally are and two of them got reworked

>  Just nerf it but I don't see an issue with the current kit.. it really isn't strong at all
- 25 damage tomahawk
- Pointman shield
- Always having armor
- Guns

**Comment by alexv-ds** [comment]
All thats left is to remove the veterans and it will be a good build

**Comment by MaslineU** [comment]
Why are you following a difficult path? It's strange to cut out an entire profession that brings to the idea of gates a difference from other servers. You could simply reduce the statistics of the operatives' equipment and open up the opportunity to play this profession to everyone. It is simple and sufficient to solve all your problems that you have recorded in the update. And so you lose the main feature of the gate. Now, as on other projects, homeless assistants will fight with unknown creatures from another multiverse. This is a bad idea.
And it is strange that in general there is a problem when it is not possible to take the gate operative to ordinary participants, since there are gate guest roles where you can go through this obstacle course with the operatives.

**Comment by Iamgoofball** [comment]
``Now, as on other projects, homeless assistants will fight with unknown creatures from another multiverse. This is a bad idea.``

the average value of an assistant: 2 credits and some duct tape
the average value of a fully kitted out vanguard operative: per this PR, 3600 credits

i think the math makes it clear which one's the more cost effective option for NT

**Comment by MaslineU** [comment]
> `Now, as on other projects, homeless assistants will fight with unknown creatures from another multiverse. This is a bad idea.`
> 
> the average value of an assistant: 2 credits and some duct tape the average value of a fully kitted out vanguard operative: per this PR, 3600 credits
> 
> i think the math makes it clear which one's the more cost effective option for NT

Haha, if you want to justify all this somehow from the Role play side, I can say that the assistants who are supposed to allegedly help the crew will not be interested in accumulating money and visiting an unknown place. They're ordinary civilians.

**Comment by MaslineU** [comment]
From the side of the game, there will be much fewer people who want to visit the gate for assistants, since they will have to accumulate money through cargo for 20 minutes to then just die in 10. I can predict that more often people will enter through ghost roles on half live gate and wait the same 20 minutes for the passage of the gate completely.

**Comment by Zonespace27** [comment]
> From the side of the game, there will be much fewer people who want to visit the gate for assistants, since they will have to accumulate money through cargo for 20 minutes to then just die in 10. I can predict that more often people will enter through ghost roles on half live gate and wait the same 20 minutes for the passage of the gate completely.

I've ran gateways several times without vanguard and this is just patently false

**Comment by TetrakoHateThis** [comment]
Is this PR implying the gateway is ever even turned on? I can't remember the last time I saw it active, so this feels sorta superfluous if the Gateway itself isn't gonna be like turned on for a majority of shifts and shifting Vanguard items to a Cargo crate just feels like the NRI crate all over again, where you're gonna see cargonians toting Vanguard Gear a whole bunch. So unless we're opening up the Gateway every shift, this feels kinda silly to fret over.

**Comment by MaslineU** [comment]
> > From the side of the game, there will be much fewer people who want to visit the gate for assistants, since they will have to accumulate money through cargo for 20 minutes to then just die in 10. I can predict that more often people will enter through ghost roles on half live gate and wait the same 20 minutes for the passage of the gate completely.
> 
> I've ran gateways several times without vanguard and this is just patently false

I did not say that it was impossible to pass the gate, I said that there would be fewer people willing to visit the gate. Be able to read

**Review by Paxilmaniac — APPROVED** [review]
Incredible. The map side seems fine.

**Comment by Zonespace27** [comment]
> shifting Vanguard items to a Cargo crate just feels like the NRI crate all over again, where you're gonna see cargonians toting Vanguard Gear a whole bunch.

Unlike the NRI crate, the contents of the vanguard crate are entirely random (3 per crate) and are a sidegrade at best to other things you can get more consistently in cargo

> Is this PR implying the gateway is ever even turned on? 

Yes (I usually do it)

**Comment by CnCForever** [comment]
Wow, hide the comments of those whose opinion does not coincide with yours at the slightest remark? How trite. In any case, I will also leave here my useless opinion. Useless due to the fact that the PR has already been released, and the coders do not know the word "no" (except for "!"). And here, in general, you don’t like ru downstream, as I see it?

My total experience of playing roles in the gate exceeds 400 hours on the ghost roles themselves and about 30-40 hours on the vanguard roles. I can say that this PR will kill gates. Why? I will write a micro-essay about this. By theses.

Firstly, the gates are a kind of "rogue-like" for the players, with a noticeable progression, somewhere better, somewhere worse, which can be seen in all gates from the "SCALAMOV" to the old Wild West. And as in any game, you will be rewarded for completing tests. Do you think that Vanguards gain strength too quickly, taking OP guns? Then what does the starting equipment have to do with it, most of which the vanguards throw away in the gate? (I’ll add on my own behalf that analogues of this equipment are obtained almost without problems at the station) Then why are you cutting out the role if the assistant who went inside the gate still leaves it in OP armor, with three machine guns and a bag of stimpacks? Maybe it's worth reducing the reward for passing the gate, or is it better to complicate it by giving a new challenge?
You shouldn’t worry about selling equipment in cargo for two reasons: there are analogues of equipment at the station, and the hype for equipment will simply disappear with time. But that's not the point. The gate can be passed without equipment in one jumpsuit and with a crowbar (normal) in hand. And if cargo technicians wear 4nr body armor and tomohawks, isn't this a job for the administration? Although how can this happen on the HRP server? (clueless)

Now let's move on to the essence of the essay. Why would this kill gates? As I like to say, it's all about motivation (another Virgil meme). In our case, both OOC motivation and IC motivation work. Players enter the round in a certain role, expecting a certain gameplay from it. Doctors enter the round knowing that they will operate, maybe even Husks, engineers know that at the beginning of the shift they will definitely have to work on the engine, assistants come in for the ERP or sit out their time in a bar / company of meta friends. Vanguard role mainers are players with server veteran status. Many of them received this status just for the role of the vanguard. They know the rules, they know that if their cannon fires at the station without the approval of the admin, they will lose their veteran status and will be banned. Most importantly, they WANT to play in the gate. They have an OOC motivation and it matches the IC motivation of their characters - that's a job for the vanguard. By removing the Vanguard role, you also remove the only profession with the motivation to explore the gate. This makes gates similar to space tide. What could be the IC-motivation of an assistant to climb into an unknown, deadly place where you can be killed, digested, disintegrated from reality, turned into a zombie or a lobotomite? Does he get paid enough for it? Are an engineer or a doctor paid to be away from the workplace with a high risk of death for several hours? No no and one more time no. The gate will only remain open to the rare antagonists who go there for OP stuff, and the occasional admin/captain approval to explore the gate. Returning to the logic "providing one vanguard costs as much as a hundred assistants" - can then we replace all the staff with assistants, giving them the knowledge and the right to do any work at the station, because it's cheaper? Maybe the division of labor, recruitment, thoughtful resource management is what makes NT a successful company? Maybe this level of development and role-playing make the difference between "Furry Tg" and "HRP server"? I don't play furry tg or even a burger station, right?

P.S. Star Trek literally tells about the work of the +expeditionary+ ship. Naturally, most of the crew should have participated in exploration and expeditions, in contrast to the crew of a stationary station with a different profile of specialization.

So... I'll stop there. I don't expect anyone to read this to the end, but I hope I got the point across. The local public prefers to extinguish any source of discontent. I will feel sorry for so many good and kind players who went in groups of vanguards to the gate, while I continue to play ghost roles. And I will also hope that you, my foreign friends, decide on which build you want to play and how to represent it. That's all. Thank you if you still read to the end. And thanks to your parents for raising their child well-mannered and able to listen to the interlocutor. You are good people. Peace for everyone.

**Comment by tf-4** [comment]
TLDR

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 1]
```diff
@@ -585,6 +600,16 @@
 	contains = list(/obj/item/mod/control/pre_equipped/loader)
 	crate_name = "cargo MODsuit crate"
 
+/datum/supply_pack/service/snowmobile
+	name = "Snowmobile kit"
+	desc = "trapped on a frigid wasteland? need to get around fast? purchase a refurbished snowmobile, with a FREE 10 microsecond warranty!"
```
Comment (tf-4) [inline modular_skyrat/modules/cargo/code/packs.dm]:
```suggestion
	desc = "Trapped on a frigid wasteland? Need to get around fast? Purchase a refurbished snowmobile with a FREE 10 microsecond warranty!"
```

**Review by tf-4 — COMMENTED** [review]
Just a quick thing before I have a think about the balance of that crate

**Comment by Iamgoofball** [comment]
``P.S. Star Trek literally tells about the work of the +expeditionary+ ship. Naturally, most of the crew should have participated in exploration and expeditions, in contrast to the crew of a stationary station with a different profile of specialization.``

gamer, deep space 9 exists and it was the best season of star trek and the inspiration for SS13 as it stands today

deep space 9 also features away teams involving groups of crewmembers despite being based on a literal space station

**Comment by HoodMilk** [comment]
Veteran here, vanguard was lame as hell and since it only had like 2 gateways designed for balance around it, so those were the only two gateways we ever went into. Their gear was insane (and looked bad). The idea of throwing some security, medics, scientists, and assistants into a gateway is much cooler.

Also they were boink bait during the job slots event due to not even having a place to use their gamer gear.

**Comment by press-mark** [comment]
packwatch

rip bozo

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm [hunk 2]
```diff
@@ -177,7 +177,7 @@
 	On top of that, your escape helicopter was shot down mid-flight, and another one won't arrive so soon; \
 	with your machinegunner being shot down with a precise headshot by something, or SOMEONE. You are likely on your own, at least for now."
 	important_text = "Keep and sustain marines' morale and discipline. Delegate responsibilities at the best of your abilities. \
-	Do not try to explore the level unless Vanguard is dead or cooperative. Stay around your area. Allowed races are humans."
```
Comment (tf-4) [inline modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:180]: why remove the bit about allowed race?

Comment (Zonespace27) [reply modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:180]: You can only see the text once you are spawned in, as a human, making this redundant

Comment (tf-4) [reply modular_skyrat/modules/black_mesa/code/ghost_spawners.dm:180]: Remove it in the other sections then

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm [hunk 3]
```diff
@@ -6,7 +6,7 @@
 	outfit = /datum/outfit/science_team
 	you_are_text = "You are a scientist in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."
 	flavour_text = "You are a scientist near the Ground Zero of a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within."
-	important_text = "Do not try to explore the level unless Vanguard is already present and close to you, and even then, wait at least 20 minutes before leaving your area. \
+	important_text = "Do not try to explore the level unless an expedition crew is already present and close to you, and even then, wait at least 20 minutes before leaving your area. \
 					Allowed races are humans."
```
Comment (Zonespace27) [inline modular_skyrat/modules/black_mesa/code/ghost_spawners.dm]:
```suggestion
	important_text = "Do not try to explore the level unless an expedition crew is already present and close to you, and even then, wait at least 20 minutes before leaving your area."
```

File: modular_skyrat/modules/black_mesa/code/ghost_spawners.dm [hunk 4]
```diff
@@ -82,7 +82,7 @@
 	You haven't heard much from the north-west post, except for the sounds of gunshots, and their radios went silent. \
 	On top of that, your escape helicopter was shot down mid-flight, and another one won't arrive so soon; \
 	with your machinegunner being shot down with a precise headshot by something, or SOMEONE. You are likely on your own, at least for now."
-	important_text = "Do not try to explore the level unless Vanguard is dead or cooperative. Stay around your area. Allowed races are humans."
+	important_text = "Do not try to explore the level unless the expedition crew is dead or cooperative. Stay around your area. Allowed races are humans."
```
Comment (Zonespace27) [inline modular_skyrat/modules/black_mesa/code/ghost_spawners.dm]:
```suggestion
	important_text = "Do not try to explore the level unless the expedition crew is dead or cooperative. Stay around your area."
```

**Comment by Stalkeros2** [comment]
Why remove the human-only reminder? Are we adding furries to Black Mesa ghost roles?

**Comment by Zonespace27** [comment]
> Why remove the human-only reminder? Are we adding furries to Black Mesa ghost roles?

No, it's just that text only appears *after* you spawn in, making it completely redundant

**Review by tf-4 — APPROVED** [review]
